src/Workleap.Extensions.Mongo/Indexing/MongoIndexer.cs

```diff
@@ -27,23 +30,22 @@ public Task UpdateIndexesAsync(Assembly assembly, string? clientName = null, str
         return this.UpdateIndexesAsync(new[] { assembly }, clientName, databaseName, cancellationToken);
     }
 
+    public Task UpdateIndexesAsync(string? clientName = null, string? databaseName = null, CancellationToken cancellationToken = default)
+    {
+        return this.UpdateIndexesAsync(Enumerable.Empty<Assembly>(), clientName, databaseName, cancellationToken);
+    }
+
     public Task UpdateIndexesAsync(IEnumerable<Assembly> assemblies, string? clientName = null, string? databaseName = null, CancellationToken cancellationToken = default)
     {
         if (assemblies == null)
         {
             throw new ArgumentNullException(nameof(assemblies));
         }
 
-        var documentTypesWithExplicitMongoCollectionAttribute = assemblies.SelectMany(x => x.GetTypes())
-            .Where(IsDocumentTypesWithExplicitMongoCollectionAttribute)
+        var documentTypes = assemblies.SelectMany(x => x.GetTypes().Where(IsDocumentTypesWithExplicitMongoCollectionAttribute))
```

Comment:
Why the rename with a less explicit name?

Reply:
I think there was an intermediary version where configurations were in there. Not sure. renamed